src/api/java/com/minecolonies/api/util/constant/WindowConstants.java

```diff
@@ -1130,6 +1130,12 @@
      */
     public static final String CITIZEN_LIVING = "living";
 
+    /**
+     * General big on and big off
+     */
+    public static final String GENERAL_ON_BIG  = LanguageHandler.format(COM_MINECOLONIES_COREMOD_GENERAL_ONBIG);
```

Comment:
We should refrain from now on from doing this.
The player can chain the language during the game without restart.
This would not be updated.

Please fix.

Reply:
Ok, I will remove all occurrences to be not constant in one go in next prs